web/public/assets/style.css

```
@@ -1,42 +1,114 @@
+  --surface:     #FFFFFF;
+  --surface2:    #F1F5F9;   /* Slate-100 */
+  --border:      #E2E8F0;   /* Slate-200 */
+  --ink:         #0F172A;   /* Slate-900 */
+  --muted:       #64748B;   /* Slate-500 */
+  --accent:      #2563EB;   /* Blue-600 */
+  --accent-text: #2563EB;
 }
 
 * { box-sizing: border-box; }
+[hidden] { display: none !important; }
 html, body { margin: 0; padding: 0; background: var(--bg); color: var(--ink); font: 16px/1.5 -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif; }
```

Stylelint (17.4.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)